JShellAPI/src/main/java/org/togetherjava/jshellapi/service/DockerService.java

```diff
@@ -29,6 +31,9 @@ public class DockerService implements DisposableBean {
 
     private final DockerClient client;
 
+    @Value("${jshell-wrapper.image-name}")
```

Comment:
Use Config object in the constructor.

Reply:
The `jshell-wrapper.image-name`  property is not under `jshellapi` in properties file

Reply:
See https://github.com/Together-Java/JShellPlaygroundBackend/pull/50#discussion_r1760314383